users/frobiac/trackpoint.h

```diff
@@ -0,0 +1 @@
+#pragma once
```

Comment:
This file should be removed.

Reply:
Based on your previous suggestions, it now contains the declaration for tp_reset() when using the userspace version, so should now be kept.